examples/preprocessing/complex_shape_3d.jl

```diff
@@ -0,0 +1,22 @@
+using TrixiParticles
+
+particle_spacing = 0.05
+
+file = "sphere"
+filename = joinpath("examples", "preprocessing", file * ".stl")
+
+# The following triangle mesh is corrupt.
+# For more robustness, use `winding_number_factor=0.4`.
+# filename = joinpath("examples", "preprocessing", "drive_gear.stl")
```

Comment:
```suggestion
filename = "sphere"

# The following triangle mesh is corrupt.
# For more robustness, use `winding_number_factor=0.4`.
# filename = "drive_gear"

file = joinpath("examples", "preprocessing", file * ".stl")
```

Reply:
This comment is obsolete. I'll remove it